src/main/resources/data/c/oregen/metals.json5

```diff
@@ -0,0 +1,137 @@
+{
+	"generators": {
```

Comment:
I think we should alphabetize these, and there's no titanium config.

Reply:
Sounds good. Things can also be split off this file arbitrarily, like gems were, so we might wind up with a misc file too for things like `coal_coke`